liwords-ui/src/chat/chat.tsx

```diff
@@ -117,7 +185,10 @@ export const Chat = React.memo((props: Props) => {
         </TabPane>
         <TabPane tab="Chat" key="CHAT">
           <div className={`chat-context${hasScroll ? ' scrolling' : ''}`}>
-            <p>{props.description}</p>
+            <p>
+              {props.description}
+              {hasUnreadChat && '*'}
+            </p>
```

Comment:
Could we use • instead of * and put a space before it, so it's more pleasant looking?

Reply:
Additionally, the math seems to not work for tablet view and maybe phone. When I scroll to the bottom after a message comes in, the indicator doesn't vanish

Reply:
Bug looks like a testing error with windows not zoomed to 100%, so way too edge case to fix. Approving.